### ISSUE TITLE
is there a way to spoof ip?

### ISSUE THREAD

**Issue by 1923812371283127398123712893126793** [zdashero/DCB#2]
so when i verify 2 accounts 1 get detected is there a countermeasure for this?

**Comment by zdashero** [comment]
Yes, use a residential proxy or vpn for the 2nd account keepin mind not every proxy or vpn will work
